gh_pages/_source/getting_started_doc.rst

```diff
@@ -0,0 +1,67 @@
+Getting Started
+===============
+
+Install ROS and Catkin
```

Comment:
`tesseract` is ROS-independent, so the implication that ROS is required to get started with `tesseract` is confusing and  a bit misleading. I would consider revising this section to clarify that ROS and `catkin` make it convenient to build `tesseract`